api/Console.json

```diff
@@ -1403,7 +1405,7 @@
               "version_added": "39"
             },
             "ie": {
-              "version_added": "11"
+              "version_added": false
```

Comment:
I think this is supported.

![image](https://user-images.githubusercontent.com/3396686/95099903-c9c6b680-0730-11eb-856e-08af851fd31a.png)

Reply:
@vinyldarkscratch did something go wrong with the methodology here? Can the other changes be trusted?

Reply:
AFAIK IE only defines console functions when DevTools is opened, so maybe that's the cause.

Reply:
If I'm correct, this was an issue of testing IE on Windows 7 vs. Windows 10 -- in BrowserStack and SauceLabs, the IE versions are ever so slightly behind on updates.  I've made sure that in the future, the Selenium scripts will always prefer Windows 10 first, and then Windows 7 if Windows 10 isn't possible.

Reply:
Does that mean IE has different behaviors in Win7 and Win10? Should that be documented here?

Reply:
It's less that IE has different behaviors, and more that the test cloud (SauceLabs) does.

Reply:
> AFAIK IE only defines console functions when DevTools is opened, so maybe that's the cause.

It sounds like maybe this wasn't the cause here, but I'll keep this in mind when reviewing any IE changes. Where would this be documented, though?